### PR TITLE
gcc: Shrink default packaging

### PR DESCRIPTION
**Pull request by jhershbe**
Shrink the size of the gcc install that is included in the base image for academic

### CODE REVIEW

File: meta/recipes-devtools/gcc/gcc-target.inc [hunk 1]
```diff
@@ -169,6 +173,7 @@ do_install () {
 
 	# We don't care about the gcc-<version> ones for this
 	rm -f *gcc-?.?*
+	rm -f *gcc-??.?*
```
Comment (amstewart) [inline meta/recipes-devtools/gcc/gcc-target.inc:176]: Is this a bug in the upstream recipe?

Comment (jhershbe) [reply meta/recipes-devtools/gcc/gcc-target.inc:176]: If you care about the space upstream, then yes

Comment (amstewart) [reply meta/recipes-devtools/gcc/gcc-target.inc:176]: OK. Well, if you had the luxury of time, I would suggest that you submit it upstream. But that isn't blocking for this review.

**Review by amstewart — APPROVED** [review]
Looks good to me. Normally, I would ask that these changes be made in the meta-nilrt layer - or else upstreamed - but we don't have time for that and there is a very low chance that this would have to be rebased to a newer OE version anyway. So this layer is fine.

**Pull request merged**